cfp_crawler/proposal/spiders/crawler.py

```diff
@@ -3,6 +3,7 @@
 import json
 from scrapy import signals
 
+
 class CrawlerSpider(scrapy.Spider):
     name = 'crawler'
```

Comment:
also can you change the name of the spider, something like cfpcrawler, cfp

Reply:
Can you confirm about this @ananyo2012 ?